src/components/shared/transactionResult/ordinary.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable complexity */
+import React, { useEffect } from 'react';
+import { getErrorReportMailto, isEmpty } from '@utils/helpers';
+import { TertiaryButton } from '@toolbox/buttons';
+import Illustration from '@toolbox/illustration';
+import getIllustration from './illustrations';
+import { txStatusTypes } from './statusConfig';
+import styles from './transactionResult.css';
+
+const errorTypes = [
+  txStatusTypes.signatureError,
+  txStatusTypes.broadcastError,
+];
+
+const Ordinary = ({
```

Comment:
can we name then `regular`? or maybe even different name, https://english.stackexchange.com/questions/23490/difference-in-usage-of-regular-usual-ordinary-normal-common looking at this both doesn't go well